app/views/teacher_mailer/deny_email.html.erb

```diff
@@ -0,0 +1,3 @@
+<p>
+  <%= sanitize @reason %>
+</p>
```

Comment:
We should probably have a linter, but this file should end with a newline. (That's what the little red icon implies.)